x-pack/plugins/security_solution/public/management/pages/policy/store/policy_list/index.test.ts

```diff
@@ -27,6 +28,7 @@ import {
   MiddlewareActionSpyHelper,
 } from '../../../../../common/store/test_utils';
 import { getPoliciesPath } from '../../../../common/routing';
+import { http } from '../../../../../../../ml/public/application/services/http_service';
```

Comment:
this is not specific to your PR. But this is a crazy import path. Nothing you can do.. but wow.

Reply:
actually - this is not suppose to be here and build is going to fail because of it 😄 

Thanks for catching it.